<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonConnect">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonActivate">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Activate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDeactivate">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Deactivate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCalibrate">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Calibrate</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>289</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
